docs/user-guide/deployments.md

```diff
@@ -221,6 +221,10 @@ template is different than `.spec.template` or if the total number of such pods
 exceeds `.spec.replicas`. It will bring up new pods with `.spec.template` if
 number of pods are less than the desired number.
 
+### History
+
+`.spec.revisionHistoryLimit` is an optional field that specifies the number of old replicasets to keep around for rollback purposes. If not specified, the history of every single rollout ever will be preserved, which is probably not what you want.
```

Comment:
I think you want to put tics around "replicaset"

I'm gonna suggest shortening the last sentence to: 
"If not specified, all replicasets will be preserved."